connectors/byoei.py

```diff
@@ -63,13 +65,14 @@ class Bulker:
     This class runs a coroutine that gets operations out of a `queue` and collects them to
     build and send bulk requests using a `client`
 
-    The bulk requests are controlled in several ways:
+    Arguments:
+
+    - `client` -- an `connectors.es.ESClient` instance
+    - `queue` -- an `asyncio,Queue` instance to pull docs from
     - `chunk_size` -- a maximum number of operations to send per request
+    - `pipeline` -- ingest pipeline settings to pass to the bulk API
     - `chunk_mem_size` -- a maximum size in MiB for each bulk request
     - `max_concurrency` -- a maximum number of concurrent bulk requests
```

Comment:
Might not relate to this PR. But I think we should have default values for fields like `chunk_size`, `chunk_mem_size` and `max_concurrency`.

Reply:
we do have defaults, they are set in the function that creates the instance, see https://github.com/elastic/connectors-python/blob/main/connectors/byoei.py#L425-L433

but maybe they could be moved at the signature level yeah. good follow-up